Library/Homebrew/diagnostic.rb

```diff
@@ -679,10 +679,17 @@ def check_git_status
 
         message = nil
 
-        {
+        taps = {
```

Comment:
```suggestion
        repos = {
```

One minor suggestion: these aren't all technically taps because `Homebrew/brew` is included, so I think `repo` is a better name.